apprise/AppriseLocale.py

```diff
@@ -219,6 +219,9 @@ def detect_language(lang=None, detect_fallback=True):
             try:
                 # Acquire our locale
                 lang = locale.getlocale()[0]
+                # Compatibility for Python >= 3.12
+                if lang == 'C':
+                    lang = 'en_US'
```

Comment:
Can you explain why this is needed?

Reply:
>  Can you explain why this is needed?

In Python 3.12, when the locale-related environment variables are unset, it will default to `C.UTF-8`, which will break [this test](https://github.com/caronc/apprise/blob/010be0bfda731dcf1449ed4814242d5de94ccb9f/test/test_apprise_translations.py#L209-L212) in Apprise.